packages/@aws-cdk/aws-lambda/test/event-source-mapping.test.ts

```diff
@@ -154,6 +154,38 @@ describe('event source mapping', () => {
     })).toThrow(/kafkaBootStrapServers must not be empty if set/);
   });
 
+  test('throws if kafkaConsumerGroupId is invalid', () => {
```

Comment:
Nice test cases for validation but let's also add some success test cases as well.

Reply:
'not throws if kafkaConsumerGroupId is valid' is the success testcase. See comment there.

Reply:
Poorly phrased on my end. What I was actually looking for here was the expected value for EventSourceMapping of success cases, but that gets covered elsewhere enough, so I'm actually happy with what you have below.